src/MizarData/include/MizarData/GetCallstackSamplingIntervals.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MIZAR_DATA_GET_CALLSTACK_SAMPLING_INTERVALS_H_
+#define MIZAR_DATA_GET_CALLSTACK_SAMPLING_INTERVALS_H_
+
+#include <stdint.h>
+
+#include <vector>
+
+#include "ClientData/CallstackData.h"
+#include "MizarBase/ThreadId.h"
+
+namespace orbit_mizar_data {
+
+// Returns a collection of time intervals in nanos between consecutive callstack samples of the same
+// thread
+[[nodiscard]] std::vector<uint64_t> GetSamplingIntervalsNs(
```

Comment:
Do we want a Duration StrongTypedef?

Reply:
We do. And it should be actually based on std::chrono. In fact, the whole way Mizar (and Orbit actually) treats time has to be reworked. But that's not a subject of this PR.